README.md

```diff
@@ -521,10 +521,12 @@ Example (always use an external build server called 'kathleen' and run make
 inside a build chroot called `chroot:lunar-amd64`). To do so, modify the
 `default_opts` sections in `~/.config/virtme-ng/virtme-ng.conf` as following:
```

Comment:
Here, the instruction suggests to "_modify the `default_opts` sections_". With that, the example was making sense.

But I don't think such file is automatically created. Then, what about:

```suggestion
inside a build chroot called `chroot:lunar-amd64`). To do so, add the
`default_opts` section in `~/.config/virtme-ng/virtme-ng.conf` as following:
```

(Not sure if, while at it, we should mention that this file can be installed in `/etc/`)

Reply:
We could also ship a `virtme-ng.conf.sample` file and mention this file.

Reply:
> We could also ship a `virtme-ng.conf.sample` file and refer to this file.

Is there not already one in `cfg/virtme-ng.conf`?

Reply:
Hmm, right. I still think it makes sense to list the full config instead of just parts of it... because a lot of people will just copy and modify it right away.